Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAuditdPlugin.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+#============================================================================
+# Copyright (C) Microsoft Corporation, All rights reserved.
+#============================================================================
+
+import os
+import sys
+import imp
+import codecs
+import subprocess
+
+protocol = imp.load_source('protocol', '../protocol.py')
+nxDSCLog = imp.load_source('nxDSCLog', '../nxDSCLog.py')
+
+LG = nxDSCLog.DSCLog
+
+PLUGIN_CONF = '/etc/audisp/plugins.d/auoms.conf'
+OMSAGENT_CONF = '/etc/opt/microsoft/omsagent/conf/omsagent.d/auoms.conf'
```

Comment:
I chose that name to match the process name, and because it isn't a generic auditd monitoring config, it is a specific auoms ingest config. I can re-name it if you really feel strongly about it, but I'd prefer to stay consistent.